src/policy_engine/opa/mod.rs

```diff
@@ -85,6 +97,21 @@ impl PolicyEngine for OPA {
 
         Ok((res_kv["allow"].as_bool().unwrap_or(false), res))
     }
+
+    async fn set_policy(&mut self, input: SetPolicyInput) -> Result<()> {
+        if input.r#type != "opa" {
+            bail!("Policy Engine Type Unmatched");
+        }
+
+        let policy_bytes = base64::decode_config(input.policy, base64::URL_SAFE_NO_PAD)
+            .map_err(|e| anyhow!("Base64 decode OPA policy string failed: {:?}", e))?;
+        let mut policy_file_path = PathBuf::from(&self.policy_dir_path.to_str().unwrap());
+        policy_file_path.push(format!("{}.rego", input.policy_id));
+
+        tokio::fs::write(&policy_file_path, policy_bytes)
```

Comment:
We can change all `std::fs` into `tokio::fs` in this file.

Reply:
Why?